Include/internal/pycore_interp.h

```diff
@@ -259,7 +259,7 @@ struct _is {
     struct callable_cache callable_cache;
     _PyOptimizerObject *optimizer;
     _PyExecutorObject *executor_list_head;
-
+    size_t executors_created;
```

Comment:
I'd suggest adding a comment saying that this number is limited to `JIT_CLEANUP_THRESHOLD`